server/app/assets/javascripts/admin_program_block_edit.ts

```diff
@@ -0,0 +1,50 @@
+import {addEventListenerToElements} from './util'
+import {init as mainInit} from './main'
+import {init as questionBankInit} from './questionBank'
+
+class AdminProgramBlockEdit {
+  public registerEventListeners() {
+    addEventListenerToElements('form.move-block', 'submit', (event: Event) =>
+      this.handlePageUpdate(event),
+    )
+
+    addEventListenerToElements('form.move-question', 'submit', (event: Event) =>
+      this.handlePageUpdate(event),
+    )
+
+    addEventListenerToElements(
+      'form.question-option-toggle',
+      'submit',
+      (event: Event) => this.handlePageUpdate(event),
+    )
+  }
+
+  private handlePageUpdate(event: Event) {
+    event.preventDefault()
+
+    const form = event.target as HTMLFormElement
+
+    fetch(form.action, {
```

Comment:
`fetch()` doesn't work in IE. Is IE one of our supported browsers?

Reply:
We've not had a formal discussion about it but I think there's other things that would break in IE on the admin side at least. IIRC the last IE release was in '13. I'd expect admins to be on at least somewhat recent versions of Windows/Edge for security reasons.

Reply:
IE is explicitly not supported: https://github.com/civiform/civiform/blob/4542c8ab50808ac4a134a8f1b9dfb18a4516791f/server/app/filters/UnsupportedBrowserFilter.java